### PR TITLE
Added search icon on user dashboard

### LINKED ISSUE #2227: Add search icon in search file - User dashboard
**Describe the bug**
The search field lacks a search icon which could help users easily identify it as a search field

**To Reproduce**
image

**Are you working on this issue? (Yes/No)**
No, a good first issue

### PR DESCRIPTION
**Pull request by Kartikay304**
Fixed #2227 

#### Describe the changes you have made in this PR - 
Added search icon & hover effect.

### Screenshots of the changes (If any) -
**Before -**
before_5sec_SparkVideo

**After -**
after_SparkVideo
@ShreyaPrasad1209, Would you please look at my PR?

### CODE REVIEW

**Comment by Kartikay304** [comment]
@ShreyaPrasad1209, Would you please look at my PR?

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -267,34 +267,30 @@
...
 }
 
-.fa.fa-search.user-circuits-search-icon {
-  color: $fa-search-grey;
+.user-circuits-search-icon {
   margin-left: 28px;
   margin-right: -28px;
   z-index: 2;
 
+  &:hover {
```
Comment (ShreyaPrasad1209) [inline app/assets/stylesheets/users.scss:290]: Changes look great but what is the need of making search bar green on hover?

**Review by ShreyaPrasad1209 — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/users.scss (hunk 1 above)
Comment (Kartikay304) [reply app/assets/stylesheets/users.scss:290]: To match search icon with CircuitVerse theme(which is green), I added hover effect on it

Comment (ShreyaPrasad1209) [reply app/assets/stylesheets/users.scss:290]: I don’t think that’s necessary. You can remove it.

Comment (Kartikay304) [reply app/assets/stylesheets/users.scss:290]: I'm sorry I was late in answering, I'll remove it right away.

**Comment by nitin10s** [comment]
@ShreyaPrasad1209 isn't this already merged in #2337?

**Pull request closed without merging**